src/main.js

```diff
@@ -107,22 +128,68 @@ var savedPosters = [];
 var currentPoster;
 
 // event listeners go here 👇
-showRandomPoster.addEventListener('click', randomPosterButton);
+showRandomButton.addEventListener('click', randomPosterButton);
+// make a poster button
+makePosterButton.addEventListener('click',hideMainShowForm);
+// saved posters sections
+savedPostersButton.addEventListener('click', showSaved);
+// go back to main buttons
+nevermindButton.addEventListener('click', backToMainPage);
+backToMainButton.addEventListener('click', backToMainPage);
+
+
 
 
 // functions and event handlers go here 👇
+
+
 // (we've provided one for you to get you started)!
 function getRandomIndex(array) {
+  // return Math.floor(Math.random() * array.length);
   return array[Math.floor(Math.random() * array.length)];
+ }
+
+
+ function randomPosterButton() {
```

Comment:
I think that we need to clean this up in the next iteration, possibly create a class here.